ncids-webpack/package.json

```diff
@@ -32,8 +32,8 @@
 		"webpack-cli": "^4.9.2"
 	},
 	"dependencies": {
-		"@nciocpl/ncids-css": "^2.3.0",
-		"@nciocpl/ncids-js": "^2.3.0"
+		"@nciocpl/ncids-css": "^3.0.0",
```

Comment:
Now run `npm install` to update the repo's `package-lock.json` file. And test that the code works using the instructions in https://github.com/NCIOCPL/ncids-example-sites/blob/ticket/6-update-to-next-ncids/ncids-webpack/README.md

Reply:
I can already tell you there is a missing breaking change for site alert, because when I updated the packages locally, the standard info site alert is borked. It is supposed to be a two column list and it is not. If the breaking change is not documented, we need to correct the release notes.

Reply:
the note on the previous PR is incorrect 
 .usa-site-alert .usa-alert__nci-list has been replaced by .usa-list.usa-list--nci-two-column (should be usa-list--two-column)

Reply:
Thank you, I have updated the documentation. 

Unfortunately, based on  our decisions in The Way, it really should have had that `nci-` prefix, but looks like we missed it.